crates/sui-indexer/src/handlers/event_handler.rs

```diff
@@ -47,13 +56,19 @@ impl EventHandler {
         loop {
             let event_page = fetch_event_page(self.rpc_client.clone(), next_cursor).await?;
             let event_count = event_page.data.len();
+            self.event_handler_metrics
+                .total_events_received
+                .inc_by(event_count as u64);
             commit_events(&mut pg_pool_conn, event_page.clone())?;
```

Comment:
+1 . You probably also want to add additional metrics between the lines such as `total_event_page_fetch`, ``total_event_page_fetch_succeed`, `total_event_page_committed`, `total_event_page_log_committed`, `total_event_page_sleep`(which is useful to tune the sleep duration)

Reply:
> is that expected or no? 

that is expected indeed, the idea is _received and _processed are expected to grow at the same speed, otherwise sth is wrong with the event processing here.

@666lcz good point, will add!